index.html

```diff
@@ -1124,49 +1124,108 @@ <h1>
+of <a>URIs</a> or objects. It is RECOMMENDED that each <a>URI</a> in the
+<code>@context</code> be one which, if dereferenced, results in a document
+containing machine-readable information about the <code>@context</code>.
+        </dd>
+      </dl>
 
-        <dl>
-          <dt><dfn>@context</dfn></dt>
-          <dd>
-The value of the <code>@context</code> property MUST be one or more
-<a>URIs</a>, where the value of the first <a>URI</a> is
-<code>https://www.w3.org/ns/did/v1</code>. If more than one
-<a>URI</a> is provided, the <a>URIs</a> MUST be interpreted as an ordered set.
-It is RECOMMENDED that dereferencing the <a>URIs</a> results in a document
-containing machine-readable information about the context.
-          </dd>
-        </dl>
+      <p>
+Though this specification requires that a <code>@context</code> <a>property</a>
+be present, it is not required that the value of the <code>@context</code>
+<a>property</a> be processed using a JSON-LD processor. This is to support
+constrained processing environments using plain JSON libraries. All libraries or
+processors MUST ensure that the order of the values in the <code>@context</code>
+<a>property</a> is what is expected for the specific application. Libraries or
+processors that support JSON-LD can process the <code>@context</code>
+<a>property</a> using full JSON-LD processing as expected.
+      </p>
 
       <p>
-Example:
+ In JSON-LD, the <code>@context</code> <a>property</a> can also be used to
+ communicate other details, such as datatype information, language information,
+ transformation rules, and so on, which are used by some DID Methods. For more
+ information, see <a href="https://www.w3.org/TR/json-ld11/#the-context">Section
+ 3.1: The Context</a> of the [[JSON-LD]] specification.
       </p>
 
-      <pre class="example nohighlight">
+      <p>
+The following example outlines a common usage of the <code>@context</code>
+<a>property</a>:
+      </p>
+
+      <pre class="example nohighlight" title="Common usage of @context property">
 {
-  "@context": "https://www.w3.org/ns/did/v1"
+  <span class="highlight">"@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://www.w3.org/ns/did/example/v1"
+  ]</span>,
+  "authentication": [{
+    "id": "did:example:123456789abcdefghi#keys-1",
+    "type": "Ed25519VerificationKey2018",
+    "controller": "did:example:123456789abcdefghi",
+    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVAjZpfkcJCwDwnZn6z3wXmqPV"
+  }]
 }
-</pre>
+      </pre>
+
+      <p>
+The example above uses the base context <a>URI</a>
+(<code>https://www.w3.org/ns/did/v1</code>) to establish that the
+conversation is about a <a>DID Document</a>. The second <a>URI</a>
+(<code>https://www.w3.org/ns/did/example/v1</code>) establishes that
+the conversation is also about the Example DID Method.
+      </p>
+
+      <p class="note">
+This document uses the example context <a>URI</a>
+(<code>https://www.w3.org/ns/did/v1</code>) for the purpose of demonstrating
```

Comment:
```suggestion
(<code>https://www.w3.org/ns/did/example/v1</code>) for the purpose of demonstrating
```
as I think the `did/v1` URI is meant to be the permanent initial `@context` value